File: Swift/프로그래머스/1/132267. 콜라 문제/콜라 문제.swift
import Foundation

func solution(_ a:Int, _ b:Int, _ n:Int) -> Int {
    var myBottle: Int = n
    
    var result: Int = 0
    
    while myBottle >= a {
        let receiveBottle: Int = (myBottle / a) * b
        result += receiveBottle
        myBottle = (myBottle % a) + receiveBottle
    }
    
    
    return result
}

File: Java/프로그래머스/1/132267. 콜라 문제/콜라 문제.java
class Solution {
    public int solution(int a, int b, int n) {
        int answer = 0;
        
        while (n >= a) {
            int receiveBottle = (n/a) * b;
            answer += receiveBottle;
            n = (n % a) + receiveBottle;
        }
        return answer;
    }
}